tests/dags/providers/provider_api_scripts/test_wikimedia_commons.py

```diff
@@ -185,54 +184,56 @@ def test_merge_image_pages_both_have_gu():
         right_page = json.load(f)
     with open(RESOURCES / "continuation/page_44672212_merged.json") as f:
         expect_merged_page = json.load(f)
-    actual_merged_page = wmc._merge_image_pages(left_page, right_page)
+    actual_merged_page = wmc.merge_image_pages(left_page, right_page)
     assert actual_merged_page == expect_merged_page
 
 
 def test_extract_title_gets_cleaned_title():
     image_info = {"extmetadata": {"ObjectName": {"value": "File:filename.jpg"}}}
-    actual_title = wmc._extract_title(image_info)
+    actual_title = wmc.extract_title(image_info)
     expected_title = "filename"
     assert actual_title == expected_title
 
     image_info["title"] = "filename.jpeg"
-    actual_title = wmc._extract_title(image_info)
+    actual_title = wmc.extract_title(image_info)
     expected_title = "filename"
     assert actual_title == expected_title
 
 
-def test_process_media_data_handles_example_dict():
+def test_get_record_data_handles_example_dict():
     """
     Converts sample json data to correct image metadata,
     and calls `add_item` once for a valid image.
     """
     with open(RESOURCES / "image_data_example.json") as f:
         media_data = json.load(f)
 
-    with patch.object(wmc.image_store, "add_item", return_value=1) as mock_add:
-        wmc._process_media_data(media_data)
+    # with patch.object(wmc.image_store, "add_item", return_value=1) as mock_add:
+    record_data = wmc.get_record_data(media_data)
+
     expected_license_info = get_license_info(
         license_url="https://creativecommons.org/licenses/by-sa/4.0"
     )
-    assert mock_add.call_count == 1
-    assert mock_add.call_args == call(
-        foreign_landing_url=(
+    # assert mock_add.call_count == 1
+    # assert mock_add.call_args == call(
```

Comment:
Did we still want to run similar checks?

Reply:
I don't think it's necessary -- checking that the records are added to the correct media store should be covered in the tests for the base class. I should've removed these after commenting them out, fixed!